unified_planning/model/problem.py

```diff
@@ -1003,6 +1010,8 @@ def update_problem_kind_action(
             if len(action.simulated_effects) > 0:
                 self.kind.set_simulated_entities("SIMULATED_EFFECTS")
             self.kind.set_time("CONTINUOUS_TIME")
+        elif isinstance(action, up.model.action.Process):
+            pass
```

Comment:
I think we should add `PROCESSES` to the kind

Reply:
yep, connected with the previous